native/src/modules/endpoint/reducers/pushCategory.js

```diff
@@ -46,6 +46,12 @@ const pushCategory = (state: CityContentStateType, action: PushCategoryActionTyp
     }
   }
 
+  // Check whether another page in the same city is loading, e.g. because it is being refreshed.
+  // This is important for displaying the loading spinner.
+  const otherPageLoading = Object.values(state.categoriesRouteMapping)
+    .filter(route => city === route.city && path !== route.path)
```

Comment:
Do we need to filter for the language as well? :thinking:

Reply:
Can't hurt I guess. Routes are probably cleared anyways when switching language.

Reply:
I was rather thinking about peeking and linking to other cities, but should not hot hurt anyway I guess.